### PR TITLE
feat: #1108 add align to ModalContentJobTags

### PR DESCRIPTION
**Pull request by AKiroi**
https://github.com/user-attachments/assets/3feec2b5-a7ee-4304-be8b-318623d18649

### CODE REVIEW

**Review by Ekaterina1994 — CHANGES_REQUESTED** [review]

File: mw-webapp/src/logic/wayPage/reportsTable/modalContentJobTags/ModalContentJobTags.module.scss [hunk 1]
```diff
@@ -1,23 +1,25 @@
...
-  display: flex;
-  flex-wrap: wrap;
-  gap: $gapSmall;
+  gap: $gapMedium;
 }
 
 .jobTags {
   display: flex;
+  flex-direction: column;
   width: $ModalJobTagMaxWidth;
```
[the remark this answers is not in the document]
Comment (AKiroi) [reply mw-webapp/src/logic/wayPage/reportsTable/modalContentJobTags/ModalContentJobTags.module.scss:16]: done

File: mw-webapp/src/logic/wayPage/reportsTable/modalContentJobTags/ModalContentJobTags.tsx [hunk 2]
```diff
@@ -80,52 +82,62 @@ export const ModalContentJobTags = observer((props: JobDoneTagsProps) => {
 
   return (
```
[the remark this answers is not in the document]
Comment (AKiroi) [reply mw-webapp/src/logic/wayPage/reportsTable/modalContentJobTags/ModalContentJobTags.tsx:83]: done

File: mw-webapp/src/logic/wayPage/reportsTable/modalContentJobTags/ModalContentJobTags.module.scss
[the remark this answers is not in the document]
Comment (AKiroi) [reply mw-webapp/src/logic/wayPage/reportsTable/modalContentJobTags/ModalContentJobTags.module.scss:1]: done

File: mw-webapp/src/logic/wayPage/reportsTable/modalContentJobTags/ModalContentJobTags.tsx [hunk 3]
```diff
@@ -80,52 +82,62 @@ export const ModalContentJobTags = observer((props: JobDoneTagsProps) => {
...
+          placeholder=""
+        />
+        <VerticalContainer className={styles.jobTagsContainer}>
+          {props.jobTags.map((tag) => {
+            return (
+              <div
+                key={tag.uuid}
+                className={styles.jobTags}
+                // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+                onClick={() => jobTagsUpdated.includes(tag.uuid) ? removeJobTagFromJobDone(tag.uuid!) : addJobTagFromJobDone(tag)}
```
[the remark this answers is not in the document]
Comment (AKiroi) [reply mw-webapp/src/logic/wayPage/reportsTable/modalContentJobTags/ModalContentJobTags.tsx:98]: done

File: mw-webapp/src/logic/wayPage/reportsTable/modalContentJobTags/ModalContentJobTags.tsx [hunk 4]
```diff
@@ -80,52 +82,62 @@ export const ModalContentJobTags = observer((props: JobDoneTagsProps) => {
...
+                  content={jobTagsUpdated.includes(tag.uuid)
+                    ? LanguageService.way.reportsTable.columnTooltip.deleteLabel[language]
+                    : LanguageService.way.reportsTable.columnTooltip.addLabel[language]
+                  }
+                  position={PositionTooltip.BOTTOM}
+                >
+                  <HorizontalContainer className={styles.labelWithCheckboxBlock}>
+                    <Label
+                      label={tag}
+                      isEditable={props.isEditable}
```
[the remark this answers is not in the document]
Comment (AKiroi) [reply mw-webapp/src/logic/wayPage/reportsTable/modalContentJobTags/ModalContentJobTags.tsx:110]: done

File: mw-webapp/src/logic/wayPage/reportsTable/modalContentJobTags/ModalContentJobTags.module.scss [hunk 5]
```diff
@@ -1,23 +1,25 @@
 @import "src/styles/variables";
 
-$ModalJobTagMaxWidth: 150px;
+$ModalJobTagMaxWidth: 300px;
```
[the remark this answers is not in the document]
Comment (AKiroi) [reply mw-webapp/src/logic/wayPage/reportsTable/modalContentJobTags/ModalContentJobTags.module.scss]: done

File: mw-webapp/src/logic/wayPage/reportsTable/modalContentJobTags/ModalContentJobTags.tsx [hunk 6]
```diff
@@ -80,52 +82,62 @@ export const ModalContentJobTags = observer((props: JobDoneTagsProps) => {
...
+                    />
+                    <Checkbox
```
[the remark this answers is not in the document]
Comment (AKiroi) [reply mw-webapp/src/logic/wayPage/reportsTable/modalContentJobTags/ModalContentJobTags.tsx]: done

**Comment by AKiroi** [comment]
mayby it will better to change "JobTags " --> "Labels" everyvere in this file? What do you think about it?

File: mw-webapp/src/dictionary/Modals.json [hunk 7]
```diff
@@ -16,6 +16,23 @@
...
+      "ru": "Сохранить",
+      "en": "Save",
+      "ua": "Зберегти"
+    }
+  },
+  "labelsModal": {
+    "title": {
+      "ru": "Выберите теги",
+      "en": "Choose tags",
+      "ua": "Виберіть теги"
```
Comment (Ekaterina1994) [inline mw-webapp/src/dictionary/Modals.json:30]: Yea? lets use labels everywhere instead jobdone

File: mw-webapp/src/logic/wayPage/reportsTable/modalContentLabels/ModalContentLabels.module.scss [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+    max-width: $descriptionMaxWidthMobile;
+  }
+
+  @media (max-width: $mediaWidthMobileMedium){
+    display: none;
+  }
+
+  p {
+    overflow: hidden;
+    text-overflow: ellipsis;
```
Comment (Ekaterina1994) [inline mw-webapp/src/logic/wayPage/reportsTable/modalContentLabels/ModalContentLabels.module.scss:53]: Are you sure that need to hide long description?

Comment (Ekaterina1994) [reply mw-webapp/src/logic/wayPage/reportsTable/modalContentLabels/ModalContentLabels.module.scss:53]: Or it's for case when user don't use spaces? If yes? then okey)

Comment (AKiroi) [reply mw-webapp/src/logic/wayPage/reportsTable/modalContentLabels/ModalContentLabels.module.scss:53]: yes, it's for case when user don't use spaces

File: mw-webapp/src/dictionary/Modals.json (hunk 7 above)
Comment (AKiroi) [reply mw-webapp/src/dictionary/Modals.json:30]: done

Comment (Ekaterina1994) [inline mw-webapp/src/dictionary/Modals.json:30]:
```suggestion
      "ru": "Выберите лэйблы",
      "en": "Choose labels",
      "ua": "Виберіть лейблы"
```

**Review by Ekaterina1994 — APPROVED** [review]

File: mw-webapp/src/dictionary/Modals.json (hunk 7 above)
Comment (AKiroi) [reply mw-webapp/src/dictionary/Modals.json]: done

**Review by tritonJS826 — APPROVED** [review]

**Pull request merged**